e2e/package-lock.json

```diff
@@ -1,1712 +1,8 @@
 {
   "name": "inventory-app-tests",
   "version": "0.0.1",
-  "lockfileVersion": 2,
```

Comment:
do we pin an npm version for the cypress tests? if we're all on v2 npm then this wouldn't revert.

Reply:
if we'd like, we can specify node/npm versions in the engines property in package.json

Reply:
https://docs.npmjs.com/cli/v7/configuring-npm/package-json#engines

Reply:
We use npm that comes from cypress docker image. But I was running a local npm command therefore the generated package-lock.json contains a unexpected lockfileVersion. I ran it again and now it is back to lockfileVersion=2.

Maybe we can ignore the file package-lock.json altogether to avoid the hassle to deal with vulnerably npm package in the future.

Reply:
If we're only using to install some libraries and we dont need to pin specific versions of transitive dependencies (which it looks like we dont) then I agree we should ignore it.